.github/workflows/_test.yml

```
@@ -5,24 +5,50 @@ on:
 permissions: {}
 
 jobs:
+  lint-test:
+    name: Lint Test
+    runs-on: ubuntu-latest
+
+    env:
+      # Set TOXENV env var to tell tox which testenv (see tox.ini) to use
+      TOXENV: lint
+
+    steps:
+      - name: Checkout TUF
+        uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c
+
+      - name: Set up Python 3.x
+        uses: actions/setup-python@d27e3f3d7c64b4bbf8e4abfb9b63b83e846e0435
+        with:
+          python-version: 3.x
+          cache: 'pip'
+          cache-dependency-path: 'requirements/*.txt'
+
+      - name: Install dependencies
+        run: |
+          python3 -m pip install --constraint requirements/build.txt tox coveralls
+
+      - name: Run tox (${{ env.TOXENV }})
+        run: tox
+
   tests:
     name: Tests
+    needs: lint-test
+    continue-on-error: ${{ matrix.failure-allowed }}
```

No need to use a variable here, if it always has the same value
```suggestion
    continue-on-error: true
```